backend/src/packages/sections/section.service.ts

```diff
@@ -183,7 +183,11 @@ class SectionService
     return {
       logo: content['logo'] ?? '',
       description: content['description'] ?? '',
-      contacts: [],
+      contacts: {
+        email: content['email'] ?? '',
+        address: content['address'] ?? '',
+        phone: content['phone'] ?? '',
+      },
```

Comment:
```suggestion
        email: content['email'] ?? '',
        address: content['address'] ?? '',
        phone: content['phone'] ?? '',
```
should be a flat object